src/style.css

```
@@ -1 +1,61 @@
-/* Styles go here */
+@import url("./general_styles.css");
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.header-logo {
+  margin: 10px 50px 7px;
+}
+
+.nav-list {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  padding: 0;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav-item {
+  height: 100%;
+  display: flex;
+  align-items: center;
+}
+
+.link {
+  text-decoration: none;
+  text-transform: uppercase;
+  color: black;
+  padding: 22.5px 0;
+  white-space: nowrap;
+  margin-right: 20px;
+  cursor: pointer;
+}
+
+.link:hover {
+  color: var(--blue);
+}
+
+.nav-item:last-child .link:last-child {
+  margin-right: 0;
+}
+
+.is-active {
+  color: var(--blue);
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  height: 4px;
+  right: 0;
+  left: 0;
+  bottom: -0.6px;
```

```suggestion
  bottom: 0;
```
Try with padding 23 instead of 22.5

---

Thank you very much, it worked :)